src/node/flows.ml

```diff
@@ -41,7 +41,7 @@ let string_of_error = function
   | `Snapshots_with_invalid_hash -> "Snapshots_with_invalid_hash"
 
 let print_error err =
-  Format.eprintf "\027[31mError: %s\027[m\n%!" (string_of_error err)
+  Logs.err (fun m -> m "\027[31mError: %s\027[m\n" (string_of_error err))
```

Comment:
Here and for other calls of Logs functions: remove `\n` from the chain, as it is automatically inserted by the library.